JavaScript/Task_Day2/MiddleCharacter.js

```diff
@@ -0,0 +1,9 @@
+//https://www.codewars.com/kata/56747fd5cb988479af000028/train/javascript
+
+//Get the Middles Character
+
+function getMiddle(str) {
+  var len = str.length;
```

Comment:
Use meaning full names, so that the readability of your code will be increased.
such as 'stringLength'.